### PR TITLE
Removing the iptalbes mangle,iptables nat and iptables as iptalbes -s…

### PR DESCRIPTION
**Pull request by suryatejaboorlu**
Removed the iptalbes mangle,iptables nat and iptables as iptalbes-save contains all these data, and also adding the 'ip neighbour' to get the ARP information

### CODE REVIEW

File: collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh [hunk 1]
```diff
@@ -244,16 +244,6 @@ networking() {
   mkdir -p $TMPDIR/networking
   iptables-save > $TMPDIR/networking/iptablessave 2>&1
   ip6tables-save > $TMPDIR/networking/ip6tablessave 2>&1
-  if [ ! "${OSRELEASE}" = "sles" ]
-    then
-      IPTABLES_FLAGS="--wait 1"
-  fi
```
Comment (leodotcloud) [inline collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh:250]: We probably need these lines.

File: collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh [hunk 2]
```diff
@@ -276,6 +266,7 @@ networking() {
   if $(command -v ip >/dev/null 2>&1); then
     ip addr show > $TMPDIR/networking/ipaddrshow 2>&1
     ip route show table all > $TMPDIR/networking/iproute 2>&1
+    ip neighbour > $TMPDIR/networking/ipneighbour 2>&1
```
Comment (leodotcloud) [inline collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh:269]: Do we need an ipv6 version of this?

**Review by leodotcloud — CHANGES_REQUESTED** [review]

File: collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh (hunk 1 above)
Comment (suryatejaboorlu) [reply collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh:250]: IPTABLES_FLAGS are only used in those 6 commands which I removed. other than that, it is not being used else where

File: collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh (hunk 2 above)
Comment (suryatejaboorlu) [reply collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh:269]: Added the ipv6 version.

Comment (leodotcloud) [reply collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh:269]: Should we have the same thing for the rest of the relevant commands too?

**Review by leodotcloud — CHANGES_REQUESTED** [review]

File: collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh (hunk 2 above)
Comment (suryatejaboorlu) [reply collection/rancher/v2.x/logs-collector/rancher2_logs_collector.sh:269]: Do you mean the IPv6 versions for the rest of the relevant commands?

**Review by leodotcloud — APPROVED** [review]

**Pull request merged**